tools/RAiDER/gnss/processDelayFiles.py

```diff
@@ -104,15 +187,61 @@ def mergeDelayFiles(
         ztdFile, 
         col_name='ZTD', 
         raider_delay='totalDelay', 
-        outName=None
+        outName=None,
+        localTime=None
     ):
     '''
     Merge a combined RAiDER delays file with a GPS ZTD delay file
     '''
     print('Merging delay files {} and {}'.format(raiderFile, ztdFile))
-
     dfr = pd.read_csv(raiderFile, parse_dates=['Datetime'])
-    dfz = readZTDFile(ztdFile, col_name=col_name)
+    dfz = pd.read_csv(ztdFile, parse_dates=['Datetime'])
+    # only pass common locations and times
+    dfz = pass_common_obs(dfr, dfz)
+    dfr = pass_common_obs(dfz, dfr)
+
+    # If specified, convert to local-time reference frame WRT 0 longitude
+    if localTime is not None:
+        from RAiDER.getStationDelays import seconds_of_day
+
+        localTime_hrs = seconds_of_day(localTime.split(' ')[0])/3600
```

Comment:
Could this be functionalized to make unit testing easier?

Reply:
If I'm understanding correctly, are you suggesting to segregate out this local-time functionality into a separate function?

Reply:
@jlmaurer done, see `local_time_filter` function